Aztec/Classes/Libxml2/DOM/Data/CommentNode.swift

```diff
@@ -35,5 +35,13 @@ extension Libxml2 {
         override func text() -> String {
             return String(.newline)
         }
+
+        override func deleteCharacters(inRange range: NSRange) {
```

Comment:
Good fix here, it looks this sorts out this crash here: https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues/365